modules/gitops-argo-cd-round-robin-sharding-algorithm.adoc

```diff
@@ -0,0 +1,235 @@
+// Module included in the following assemblies:
+//
+// * declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argo-cd-round-robin-sharding-algorithm_{context}"]
+= Enabling the round-robin sharding algorithm
+
+:FeatureName: The `round-robin` sharding algorithm
+include::snippets/technology-preview.adoc[]
+
+By default the Argo CD Application Controller uses the non-uniform `legacy` hash-based sharding algorithm to assign clusters to shards. This can result in uneven cluster distribution. You can enable the `round-robin` sharding algorithm to achieve more equal cluster distribution across all shards.
+
+Using the `round-robin` sharding algorithm in the {gitops-title} provides the following benefits:
+
+* Ensure more balanced workload distribution
+* Prevent shards from being overloaded or underutilized
+* Optimize the efficiency of computing resources
+* Reduce the risk of bottlenecks
+* Improve overall performance and reliability of the Argo CD system
+
+The introduction of alternative sharding algorithms allows for further customization based on specific use cases. You can select the algorithm that best aligns with your deployment needs, which results in greater flexibility and adaptability in diverse operational scenarios.
+
+[TIP]
+====
+To leverage the benefits of alternative sharding algorithms in OpenShift GitOps, it is crucial to enable sharding during deployment.
+====
+
+[id="enabling-the-round-robin-sharding-algorithm-in-web-console_{context}"]
+== Enabling the round-robin sharding algorithm in web console
+
+You can enable the `round-robin` sharding algorithm using the {OCP} web console.
```

Comment:
```suggestion
You can enable the `round-robin` sharding algorithm by using the {OCP} web console.
```
See guidance on "using" in the [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u) guide.